pkg/healthz/healthz_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package healthz_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"testing"
+)
+
+func TestHealthz(t *testing.T) {
```

Comment:
They're the minimal test files that ginkgo initializes. We don't gain a lot initially, but we won't forget to initialize ginkgo tests in the future.